binary-search/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.flake8Enabled": false
+}
```

Comment:
Please delete this file as it is unnecessary and is just generated from VSCode. Then add your OS and 'Code' to your `.gitignore` using gitignore.io, that should make sure this doesn't pop up again.